src/index.js

```diff
@@ -0,0 +1,155 @@
+};
+const showCommentsInfo = async (event) => {
+  const button = event.target;
+  const mealId = button.id;
+  const mealInfo = await getMealInfo(mealId);
+  if (mealInfo !== undefined) {
+    const meal = Object.values(mealInfo.meals)[0];
+
+    const { countComments, comments } = await getCommentsById(mealId);
+    let commentsSection = '';
+    if (countComments > 0) {
+      comments.forEach((comment) => {
+        commentsSection += `<li>${comment.creation_date} ${comment.username}: ${comment.comment}</li>`;
+      });
+    }
+    const popupContainer = document.createElement('div');
+    popupContainer.classList.add('popupContainer');
+    const popup = document.createElement('div');
+    popup.classList.add('popup');
+    popup.innerHTML = `<div class='heading'> <a class='close' href='#'><img src='${close}' alt='close icon'></a></div>
+  <div class="content"><div ><div><img class="dishPop" src="${meal.strMealThumb}" alt="mealImage"></div><h1>${meal.strMeal}</h1><p class='instructions' >${meal.strInstructions}</p></div>
+  <h2>Comments(${countComments})</h2>
+  <ul class='comments'>${commentsSection}</ul>
+  <form>
+  <h2>Add a comment</h2>
+  <input type="text" name="name" id="name" placeholder="Your name" required>
+  <textarea  id="insights" class="area"  name="insights" rows="5" cols="15" maxlength="500" placeholder="Your insights" required></textarea>
+  <input class="newButton" type="submit" value="Comment">
+  <input type="hidden" name="mealId" value="${meal.idMeal}">
+</form>
+  </div>`;
+
+    popupContainer.appendChild(popup);
+    document.body.prepend(popupContainer);
+    const linkClose = document.getElementsByClassName('close')[0];
+    linkClose.addEventListener('click', closePopup);
+    const form = document.getElementsByTagName('form')[0];
+    form.addEventListener('submit', submitComment);
+  }
+};
+
+const getInitialData = async () => {
+  let content = '';
+  let mealsCount = 0;
+  const meals = await getMeals(MEALS_CATEGORY);
+  if (meals !== undefined) {
+    const likes = await getInvolvementData();
+    const data = Object.values(meals)[0];
+    mealsCount = data.length;
+    for (let index = 0; index < mealsCount; index += 1) {
+      const meal = data[index];
+      const like = likes.filter((l) => l.item_id === meal.idMeal);
+      const likesCount = (like.length === 0) ? 0 : like[0].likes;
+      const itemTemplate = `<div class='meal'><img class='dish' src='${meal.strMealThumb}' alt='${meal.strMeal} image' ><div class='description'><p>${meal.strMeal}</p><div class='likes'><input type="hidden" name="likesCount" value="${likesCount}"><img class='heart' id='${meal.idMeal}' src='${whiteHeart}' alt='likeImg' ></img><label for=''>${likesCount} likes</label></div></div><input class='newButton' type='submit' id='${meal.idMeal}' name='CommentsButton' value='Comments'> <input class='newButton' type='submit' id='${meal.idMeal}' name='ReservationsButton' value='Reservations'></div>`;
```

Comment:
- [ ] **Required:** You have done an amazing design so far. Kudos. 👏 However, there are small changes to look similar to the design template as follows: 

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/164944014-550c42d2-be1d-4ff5-ae44-4c35fe971b19.png) | ![image](https://user-images.githubusercontent.com/27709832/164944037-64d82ecf-f414-4248-b6ea-f1f78385ba01.png)

- Increase the size of the like icon and the "likes" words as well.
- Kindly centre the buttons as shown in the template

Reply:
- [ ] [OPTIONAL] Great implementation @NeckerFree however, you can make it better. Let's reduce the size of the like icon for a better UI.
![Screenshot from 2022-04-23 23-48-45](https://user-images.githubusercontent.com/58692590/164948714-cadefe53-0268-4407-8719-a2f7319f6d2b.png)